eventually/eventually/settings.py

```diff
@@ -24,7 +24,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['*']
```

Comment:
for what do you commit that?

Reply:
i really need this shit)))

Reply:
please describe the importance of this change.

Reply:
I have a nginx web server raised, this star gives him access to the host. Why is she here?) I do not think it would be advisable to cut off separate branches for one star

Reply:
You can keep this change only locally on your machine and when wi will have the vps change this field to the ip of that vps.

Reply:
@dmytrorevak You do not know those who speak a lot, do not live for a long time?)))))